prow/gerrit/adapter/adapter.go

```diff
@@ -310,11 +313,30 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 
 	changedFiles := listChangedFiles(change)
 
+	//Attempt to Get InRepoConfig
+	opts := git.ClientFactoryOpts{
+		CloneURI:       cloneURI.String(),
+		Host:           cloneURI.Host,
+		CookieFilePath: c.cookieFilePath,
+	}
+	gc, err := git.NewClientFactory(opts.Apply)
```

Comment:
while we don't have integration test, I think we'll need to set up unit test for this, you can use the same local git at https://github.com/kubernetes/test-infra/blob/e84d71dcb03994718ffe9e7d637acf0ae080b0b2/prow/config/inrepoconfig_test.go#L559 for unit testing this

Reply:
I thought I would not be able to at first because the code to get the prow yaml is too complicated to include in the Unit test for the adapter, but I forgot that the ProwYamlGetter can be mocked. Went back and added unit tests